src/store/courses/reducer.js

```diff
@@ -0,0 +1,45 @@
+import { ADD_COURSE, DELETE_COURSE, SAVE_COURSES } from './actionTypes';
+
+const coursesInitialState = [];
+
+export default function coursesReducer(
+	state = coursesInitialState,
+	{ type, payload }
+) {
+	switch (type) {
+		case SAVE_COURSES: {
+			return payload;
+		}
+		case ADD_COURSE: {
```

Comment:
ADD_COURSE is basically SAVE_COURSE, see the screen above

Reply:
Changed